pkg/phlare/phlare.go

```diff
@@ -200,6 +202,11 @@ func (c *Config) RegisterFlagsWithContext(f *flag.FlagSet) {
 	c.API.RegisterFlags(f)
 	c.EmbeddedGrafana.RegisterFlags(f)
 	c.TenantSettings.RegisterFlags(f)
+
+	c.v2Experiment = os.Getenv("PYROSCOPE_V2_EXPERIMENT") != ""
```

Comment:
don't we already have `c.v2Experiment = os.Getenv("PYROSCOPE_V2_EXPERIMENT") != ""`  in `registerServerFlagsWithChangedDefaultValues` , do we need this duplication?